go/theatre/testdata/TestPrinterPrintByApproval/1.in.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Any chance this could be named after the actual test case ie "TestPrinterPrintByApproval" rather than "1"?

Reply:
The folder this file is in is named after the test, and the approval function iterates over all *.in.json files. The function supports a table-like approach where you could add more tests for the same test procedure.
For example, testing the error result (for unknown genres for instance), a separate folder would be made.